### ISSUE TITLE
Separately color header, body and footer in RadialTimePicker

### ISSUE THREAD

**Issue by bridgeri127** [code-troopers/android-betterpickers#291]
I wand to use the library for a radial time picker, but I'm getting blocked by trying to color separate elements of the dialog differently. I am aiming for this: 
<img width="275" alt="screen shot 2016-05-24 at 15 50 49" src="https://cloud.githubusercontent.com/assets/9522610/15508438/531a2ede-21c7-11e6-8d88-5fe2aae48822.png">

but the closest I can get is:
<img width="278" alt="screen shot 2016-05-24 at 15 51 24" src="https://cloud.githubusercontent.com/assets/9522610/15508451/60d5f8fa-21c7-11e6-8d5d-d98887ee7c64.png">

I see that the following lines of code are responsible for setting the background color and define each of them as having the same color. Is it possible to over ride this and color them independently?

```
// Prepare some colors to use.
        int mainColor1 = themeColors.getColor(R.styleable.BetterPickersDialog_bpMainColor1, R.color.bpWhite);
        int mainColor2 = themeColors.getColor(R.styleable.BetterPickersDialog_bpMainColor2, R.color.circle_background);
        int lineColor = themeColors.getColor(R.styleable.BetterPickersDialog_bpLineColor, R.color.bpLine_background);
        int mainTextColor = themeColors.getColor(R.styleable.BetterPickersDialog_bpMainTextColor, R.color.numbers_text_color);

        // Set the colors for each view based on the theme.
        view.findViewById(R.id.time_display_background).setBackgroundColor(mainColor1);
        view.findViewById(R.id.ok_cancel_buttons_layout).setBackgroundColor(mainColor1);
        view.findViewById(R.id.time_display).setBackgroundColor(mainColor1);
        ((TextView) view.findViewById(R.id.separator)).setTextColor(mainTextColor);
        ((TextView) view.findViewById(R.id.ampm_label)).setTextColor(mainTextColor);
```

**Comment by bridgeri127** [comment]
I was able to work this out eventually, by extending the class and addressing those views directly myself. However, this would have been made much easier if the fields of the class were protected instead of private, do you think changing the access modifiers is something that could happen in a future release? It would make customising and adapting the library so much easier.

**Comment by fchauveau** [comment]
Yes we can change visibility of fields.
But we certainly must change display of radial to make customization easier

**Comment by fchauveau** [comment]
@bridgeri127 white part of header is fixed in release 2.5.6

**Comment by fchauveau** [comment]
Please see v3.0.0 release that offer new styling features
